distribution/cli/distribute.go

```diff
@@ -0,0 +1,52 @@
+package cli
```

Comment:
Reminder to delete this file from jfrog-cli after the lifecycle commands are migrated.
I suggest moving it to a share utils packages since it used by both distribution and lifecycle

Reply:
This file has a duplication in jfrog-cli/utils/distribution/distribute.go which wasn't removed in the jfrog-cli PR.
The lifecycle package in jfrog-cli should use the file in this repository and the duplication should be deleted.